### PR TITLE
update licence headers via script

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @DeepDiver1975 @LukasReschke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
I also updated the script to only use the code and not the license header for the attribution detection ;)

File: apps/files_external/lib/sftp.php [hunk 1]
```diff
@@ -7,7 +7,9 @@
  * @author Lennart Rosam <lennart.rosam@medien-systempartner.de>
  * @author Lukas Reschke <lukas@owncloud.com>
  * @author Morris Jobke <hey@morrisjobke.de>
+ * @author Robin Appelman <icewind@owncloud.com>
  * @author Robin McCorkell <rmccorkell@karoshi.org.uk>
+ * @author Robin McCorkell <rmccorkell@owncloud.com>
```
Comment (DeepDiver1975) [inline apps/files_external/lib/sftp.php]: @MorrisJobke mailmap needs to be updated to not list @Xenopathic twice

Comment (MorrisJobke) [inline apps/files_external/lib/sftp.php]: Will do so

**Comment by DeepDiver1975** [comment]
:+1:

File: apps/files_external/lib/sftp.php (hunk 1 above)
Comment (MorrisJobke) [inline apps/files_external/lib/sftp.php]: 8366ce2

**Comment by rullzer** [comment]
:+1:

**Pull request merged**